.templates/zlib.tmpl

```diff
@@ -0,0 +1,19 @@
+zlib License
+
+C {{.Year}} {{.Name}}
```

Comment:
https://www.zlib.net/zlib_license.html

The license text at the above URL uses the full word "Copyright". Can you make this change?